libcudacxx/include/cuda/std/detail/libcxx/include/__cuda/ptx.h

```diff
@@ -31,7 +31,7 @@
 #include "../__cuda/ptx/ptx_helper_functions.h"
 #include "../__cuda/ptx/parallel_synchronization_and_communication_instructions_mbarrier.h"
 #include "../cstdint" // uint32_t
-
+#include "../__type_traits/is_same.h" // is_same
```

Comment:
I did not find it in the diff, can this go?

Reply:
Yes good catch. I will remove it.